src/libs/ValidationUtils.js

```diff
@@ -28,6 +28,9 @@ function isRequiredFulfilled(value) {
     if (_.isString(value)) {
         return !_.isEmpty(value.trim());
     }
+    if (_.isDate(value)) {
+        return moment(value).isValid();
```

Comment:
Hm, I wonder if we should use `isValidDate` here or not?

Reply:
Seems like a good idea to use `isValidDate` here b/c DRY, but NAB because `isValidDate` only catches extreme fringe cases beyond this.